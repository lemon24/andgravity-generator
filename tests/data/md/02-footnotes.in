one[^1]

[^1]: o-n-e

two[^word][^1]

[^word]: t-w-o

