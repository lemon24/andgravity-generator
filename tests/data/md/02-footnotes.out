<p>one<sup class="footnote-ref" id="fnref-1"><a href="#fn-1">1</a></sup></p>
<p>two<sup class="footnote-ref" id="fnref-2"><a href="#fn-2">2</a></sup><sup class="footnote-ref" id="fnref-3"><a href="#fn-3">3</a></sup></p>
<section class="footnotes">
<ol>
<li id="fn-1"><p>o-n-e<a href="#fnref-1" class="footnote">&#8617;</a></p></li>
<li id="fn-2"><p>t-w-o<a href="#fnref-2" class="footnote">&#8617;</a></p></li>
<li id="fn-3"><p>o-n-e<a href="#fnref-3" class="footnote">&#8617;</a></p></li>
</ol>
</section>
